fusion-cli/build/get-webpack-config.js

```diff
@@ -398,12 +395,29 @@ function getWebpackConfig(opts /*: WebpackConfigOpts */) {
       runtime === 'server' &&
         ((context, request, callback) => {
           if (/^[@a-z\-0-9]+/.test(request)) {
-            // do not bundle external packages and those not whitelisted
             const absolutePath = resolveFrom.silent(context, request);
+            // do not bundle external packages and those not whitelisted
             if (absolutePath === null) {
               // if module is missing, skip rewriting to absolute path
               return callback(null, request);
             }
+            if (experimentalCompileTest) {
+              const {bundle} = experimentalCompileTest(absolutePath, {
+                transform: 'none', // default transform value doesn't actually matter here
```

Comment:
It seems slightly odd that we don't use the returned transform value at all (and instead just re-invoke the function inside the babel config setup). I wonder if it should just be two functions then (one for bundle value, the other for transform value). Alternatively, it might make sense to cache/memoize the function so it doesn't get re-invoked multiple time.